public/app/core/components/PermissionList/PermissionList.tsx

```diff
@@ -24,7 +24,6 @@ class PermissionList extends PureComponent<Props> {
             item={{
               name: 'Admin',
               permission: 4,
-              icon: 'fa fa-fw fa-street-view',
```

Comment:
As you can see in code We are[ passing icon to DisabledPermissionsListItem](https://github.com/grafana/grafana/blob/master/public/app/core/components/PermissionList/PermissionList.tsx#L22-L29), but in [DisabledPermissionsListItem](https://github.com/grafana/grafana/blob/master/public/app/core/components/PermissionList/PermissionList.tsx#L22-L29), we are not using it. Therefore it can be removed.